tools/issue-labeler/src/IssueLabeler/Program.cs

```diff
@@ -0,0 +1,18 @@
+using Microsoft.Azure.Functions.Worker;
```

Comment:
We'll want to add the common header to all files.    It's a bit sporradic currently.